sw/device/silicon_creator/lib/drivers/hmac.h

```diff
@@ -35,10 +35,22 @@ typedef struct hmac_digest {
  * Initializes the HMAC in SHA256 mode.
  *
  * This function resets the HMAC module to clear the digest register.
+ * It then configures the HMAC block in SHA256 mode with digest output
+ * in the requested endianness.
+ *
+ * @param big_endian Whether or not to initialize the peripheral for big-endian
+ *                   results.
+ */
+void hmac_sha256_init_endian(bool big_endian);
```

Comment:
nit: I'd specify digest in the parameter name, since the HMAC block also has configuration parameters for key and message endianness that we don't touch here.
```suggestion
void hmac_sha256_init_endian(bool big_endian_digest);
```